Blackjack/Other/NSMutableArray_Shuffling.h

```diff
@@ -0,0 +1,18 @@
+//
+//  NSMutableArray_Shuffling.h
```

Comment:
Category names are with a '+'.

In this case - NSMutableArray+Shuffling.h

Reply:
Done